Logger/FileLogger.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.IO;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        private string? _filePath;
+        public string? FilePath
+        {
+            get
+            {
+                return _filePath;
+            }
+            set => _filePath = value;
+        }
+        public FileLogger(string filePath)

```

Comment:
```suggestion
        public FileLogger(string? filePath)
```
This should be nullable if you're throwing a null exception below.
Setting it to nullable will also allow you to remove the `!` operator in your unit tests.